enterprise/internal/codeintel/stores/dbstore/indexes.go

```diff
@@ -323,7 +323,7 @@ FROM lsif_indexes_with_repository_name u
 LEFT JOIN (` + indexRankQueryFragment + `) s
 ON u.id = s.id
 JOIN repo ON repo.id = u.repository_id
-WHERE %s ORDER BY queued_at DESC LIMIT %d OFFSET %d
+WHERE %s ORDER BY queued_at, u.id DESC LIMIT %d OFFSET %d
```

Comment:
```suggestion
WHERE %s ORDER BY queued_at DESC, u.id LIMIT %d OFFSET %d
```

This should preserve the tests (and actually be what we want to see). What I told you to add here was wrong because what we want is newest first, but the `DESC` only applies to the `u.id` here.